utils/tests/test_unauthenticated_endpoints.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+import pytest
+from django_extensions.management.commands import show_urls
+import json
+import requests
+from lxml import html
+import mvj.urls
+
+
+def paths():
+    cmd = show_urls.Command()
+    views_str = cmd.handle(
+        no_color=True,
+        language=None, decorator=None,
+        format_style='pretty-json',
+        urlconf="ROOT_URLCONF",unsorted=True)
+    views = json.loads(views_str)
+    return [path for view in views if (path := view.get('url'))]
+
+repls = (('/<pk>\.<format>/','/1234.json/'),
+         ('/<pk>/','/1234/'),
+         ('/<path:object_id>/','/abc/123/'))
+
+loginpagetitle = 'Kirjaudu sisään | Django-sivuston ylläpito'
+
+whitelist = '''
+    /auth/login/
+    /auth/logout/
+    /v1/pub/answer/
+    /v1/pub/faq/
+    /v1/pub/form/
+    /v1/pub/plot_search_ui/0
+    /v1/pub/plot_search/
+    /v1/pub/plot_search_stage/
+    /v1/pub/plot_search_type/
+    /v1/pub/plot_search_subtype/
+    /v1/pub/plot_search_ui/1
+    /docs/
+    /redoc/
+''' .split()
+
+@pytest.mark.parametrize('path',paths())
+def test_urls(path):
+    if not path.endswith('/') or path in whitelist: pytest.skip()
+    for r in repls:
+        path = path.replace(*r)
+    if '>' in path: pytest.skip() # This includes lone '\.<format>' and few corner cases
+    url = f'http://localhost:8001{path}'
+    res = requests.get(url)
+    ct = res.headers.get('content-type')
```

Comment:
This is mostly about preference, but I don't see any advantage shortening variable names so that it isn't apparent what they are. For example I would use `content_type` for this, shortening it doesn't seem beneficial.

Also the line above has `res` which could easily be response, even though res is often used, not everybody might instantly know what it is.
Same goes for the line below, sc -> `status_code`